src/buffer/buffer.rs

```diff
@@ -93,19 +94,37 @@ impl Buffer {
     }
 
     /// Returns a reference to Cell at the given coordinates
+    /// # Panics
+    /// Panics if the coordinates are out of bounds
     #[track_caller]
     pub fn get(&self, x: u16, y: u16) -> &Cell {
         let i = self.index_of(x, y);
         &self.content[i]
     }
 
+    /// Returns an option containing a reference to Cell at the given coordinates, or `None`, if the
+    /// coordinates are out of bounds
+    pub fn try_get(&self, x: u16, y: u16) -> Option<&Cell> {
```

Comment:
Should be called get_opt as `try_` indicates a method will return `Result`. We briefly discussed naming in the underlying issue.